features/shipping/managing_shipments/filtering_shipments_by_channel.feature

```diff
@@ -0,0 +1,37 @@
+@managing_shipments
+Feature: Filtering shipments by channel
+    In order to filter shipments by a specific channel
```

Comment:
```suggestion
    In order to browse only relevant shipments from a specific channel
```
Not sure about this construction, but I was trying to remove `filter` duplication.